src/js/components/Calendar/__tests__/Calendar-test.tsx

```diff
@@ -28,6 +28,12 @@ describe('Calendar', () => {
     expect(results).toHaveNoViolations();
   });
 
+  test('date', () => {
```

Comment:
```suggestion
  test('render without grommet wrapper', () => {
```
Can we just rename this test?